frontend/package.json

```diff
@@ -23,8 +23,8 @@
   "scripts": {
     "build:css": "postcss src/tailwind.css -o src/index.css",
     "watch:css": "postcss src/tailwind.css -o src/index.css",
-    "start": "yarn watch:css && react-scripts start",
-    "build": "yarn build:css && react-scripts build",
+    "start": "npm run watch:css && react-scripts start",
+    "build": "npm run build:css && react-scripts build",
```

Comment:
We using yarn my dude

Reply:
The difference is that this works whether you use yarn or npm, while the prior one only works if you use yarn